container.go

```diff
@@ -43,3 +43,18 @@ func (c *Client) ContainersStop(ctx context.Context, appName, containerID string
 
 	return nil
 }
+
+func (c *Client) ContainersSendSignal(ctx context.Context, app string, signal string, container string) error {
```

Comment:
naming: I'm not completely convinced by the variable name `container`. Is it a name (e.g. `web-1`) or an ID (e.g. `5c9b95d201dcea2f34d496a3`)? Depending on the answer I would name it `containerName` or `containerID`.

Reply:
It's a `containerName`, reasons why it's a `docker_name` instead of a `docker_id` is explained here https://github.com/Scalingo/api/pull/2096#discussion_r1057484461